HW2- SQL File

```diff
@@ -0,0 +1,78 @@
+/*-- BDA 696 HW 2 --
+ * Karenina Zaballa
+
+
+INSTRUCTIONS:
+
+GOAL: Calculate the batting average using SQL queries for every player
+
+ 1. Start with one player
+ 		Do Annual
+ 		Do Historic
+ 		Rolling (over last 100 days) ? - Look at the last 100 days that player was in prior to this game
+ 2. Apply above to all players
+   Store all these results in their own tables
+*/
+
+show errors;
```

Comment:
Why is this line and line 18 here? it didnt do anything for me.

Reply:
I thought it would explicitly show my errors. For now, it's not that important